packages/code-du-travail-modeles/src/build.ts

```diff
@@ -33,5 +35,21 @@ function copyJSFile() {
   fse.copy(`${inDir}/utils`, `${outDir}/utils`);
 }
 
+function writeSupportedCCFile() {
+  const modeles = mergeModels();
+  const ccn = extractImplementedCc(new Engine(modeles));
+  const jsString =
+    '"use strict";\n' +
```

Comment:
Could it be a json file ?

Reply:
I prefer using typescprit here to have properties types